drivers/overlay/overlay.go

```diff
@@ -400,11 +400,11 @@ func supportsOverlay(home string, homeMagic graphdriver.FsMagic, rootUID, rootGI
 			_ = os.RemoveAll(layerDir)
 			_ = os.Remove(home)
 		}()
-		_ = idtools.MkdirAs(mergedDir, 0700, rootUID, rootGID)
-		_ = idtools.MkdirAs(lower1Dir, 0700, rootUID, rootGID)
-		_ = idtools.MkdirAs(lower2Dir, 0700, rootUID, rootGID)
-		_ = idtools.MkdirAs(upperDir, 0700, rootUID, rootGID)
-		_ = idtools.MkdirAs(workDir, 0700, rootUID, rootGID)
+		_ = idtools.MkdirAs(mergedDir, 0500, rootUID, rootGID)
+		_ = idtools.MkdirAs(lower1Dir, 0500, rootUID, rootGID)
+		_ = idtools.MkdirAs(lower2Dir, 0500, rootUID, rootGID)
+		_ = idtools.MkdirAs(upperDir, 0500, rootUID, rootGID)
+		_ = idtools.MkdirAs(workDir, 0500, rootUID, rootGID)
```

Comment:
This is run-time feature test code.  We delete these directories when the test is finished.  What's the purpose of this hunk?

Reply:
Just globally changed the permissions.